django-stubs/db/migrations/operations/base.pyi

```diff
@@ -1,18 +1,28 @@
 from collections.abc import Sequence
-from typing import Any
+from enum import Enum
+from typing import Any, ClassVar
 
 from django.db.backends.base.base import BaseDatabaseWrapper
 from django.db.backends.base.schema import BaseDatabaseSchemaEditor
 from django.db.migrations.state import ProjectState
 from django.db.models import Model
 from typing_extensions import Self
 
+class OperationCategory(str, Enum):
+    ADDITION = "+"
+    REMOVAL = "-"
+    ALTERATION = "~"
+    PYTHON = "p"
+    SQL = "s"
+    MIXED = "?"
```

Comment:
Having the actual values seems to be [recommended](https://typing.readthedocs.io/en/latest/guides/writing_stubs.html#classes).